core/logger/Logger.cpp

```diff
@@ -267,14 +286,32 @@ void Logger::LoadConfig(const std::string& filePath) {
         logConfigInfo = "Load log config from " + filePath;
     } while (0);
 
+    // parse env log level
+    level::level_enum* envLogLevel = new(level::level_enum);
+    std::string aliyun_logtail_log_level = "";
+    std::string logLevelInfo;
+    if (ReadLogLevelFromEnv(envLogLevel, aliyun_logtail_log_level)) {
+        if (!aliyun_logtail_log_level.empty()) {
+            logLevelInfo = "read log level from the env successfully, level: " + aliyun_logtail_log_level;
+        }
+        for (auto& loggerConfig : loggerConfigs) {
+            loggerConfig.second.level = *envLogLevel;
```

Comment:
please only modify LogLevel if its config name is "/apsara/sls/ilogtail"

Reply:
We do not need to modify the log level in 3 places. We can change it at load time: logger->flush_on(loggerCfg.level);
As we do not want the debug config to be persisted, this is not a bad idea.

Reply:
Ok, I will fix them.